scripts/exp/update-internal-dep-versions.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { projectRoot } = require('../utils');
+const { mapPkgNameToPkgJson } = require('../release/utils/workspace');
+const { argv } = require('yargs');
+const fs = require('mz/fs');
+
+async function updateField(pkg, fieldName) {
+  const field = pkg[fieldName];
+  for (const depName in field) {
+    if (!depName.includes('@firebase')) continue;
```

Comment:
We should update the `firebase` package as well as packages may depend on it, e.g.`@firebase/testing` and `rxFire`

Reply:
Updated.